target/linux/mvebu/cortexa9/base-files/lib/preinit/81_linksys_syscfg

```diff
@@ -8,12 +8,7 @@ preinit_mount_syscfg() {
 	. /lib/upgrade/common.sh
 
 	case $(board_name) in
-	linksys,wrt1200ac|\
-	linksys,wrt1900ac-v1|\
-	linksys,wrt1900ac-v2|\
-	linksys,wrt1900acs|\
-	linksys,wrt3200acm|\
-	linksys,wrt32x)
+	linksys,wrt1900ac-v1)
```

Comment:
If you like, just add the space before "cut" two lines below for this file as well.